beacon-chain/rpc/proposer/server.go

```diff
@@ -356,6 +357,49 @@ func (ps *Server) defaultEth1DataResponse(ctx context.Context, currentHeight *bi
 	}, nil
 }
 
+// This filters the input attestations to return a list of valid attestations to be packaged inside a beacon block.
+func (ps *Server) filterAtts(ctx context.Context, slot uint64, atts []*ethpb.Attestation) ([]*ethpb.Attestation, error) {
```

Comment:
Naming could be clearer - something like `filterAttestationsReadyForInclusion`. Would rather be more verbose in naming these RPC helper funcs for future reference.